src/web/src/mixins/NotificationsMixin.js

```diff
@@ -10,27 +11,41 @@ import { getBackgroundColor, getBorderColor } from "@/services/ColorService";
 export default {
   methods: {
     /**
-     * Generates a Bootstrap Toast notification of a schedule conflict
-     * using the provided information
-     * @param {Course} course
-     * @param {CourseSession} conflictSession
-     * @param {Subsemester} subsemester
+     * Generates a Bootstrap Toast Notification of a Schedule Conflict
+     * Using The Provided Information, Specified Below:
+     * @param {Course} addCourse = Course Tried To Add,
+     * @param {Course} existCourse = Existing Conflict Course,
+     * @param {CourseSession} addSession = Course Session Data For Attemped Course.
+     * @param {CourseSession} existSession = Course Session Data For Existing Course.
      */
-    notifyScheduleConflict(course, conflictSession, subsemester) {
+    notifyScheduleConflict(addCourse, existCourse, addSession, existSession) {
+      //NOTE: If Sub-Semester Data Desired, Need To Add As Parameter +
+      //Pass From CourseScheduler.vue.
+      // const vNodesMsg = this.$createElement("p", { class: ["mb-0"] }, [
+      //   `${subsemester.display_string}: Conflict With ${existSession.crn}
+      //         - ${existSession.section} `,
+      //Format Time Into Human-Readable Via Moment (e.g., 10:00 AM):
```

Comment:
No need to explain what moment.format does, this is implied in the method name.

Generally I'd prefer to stay away from these single line comments that explain what the next line does, usually adds more lines of comments without adding much value as long as the method names are descriptive enough.

What do you think?